libc/stdio_file.c

```diff
@@ -77,6 +88,8 @@ int checkdescriptor(FILE *filep)
 
 void assert_fopen_error(char *path, char *opts, int errnocode)
```

Comment:
static, applies to other local functions also, `const char *path`, `const char *opts`